ironfish/src/network/messages/signalRequest.ts

```diff
@@ -0,0 +1,51 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+import bufio from 'bufio'
+import { Identity } from '../identity'
+import { NetworkMessage, NetworkMessageType } from './networkMessage'
+
+interface CreateSignalRequestMessageOptions {
+  destinationIdentity: Identity
+  sourceIdentity: Identity
+}
+
+/**
+ * A message used to indicate to a peer that we want them to
+ * initiatie signaling with us. This is most often used when
```

Comment:
Nit: Looks like this was here prior to this PR, but `initiatie` -> `initiate`.

Reply:
Thank you sir